addons/website/static/src/snippets/s_embed_code/options.js

```diff
@@ -0,0 +1,86 @@
+odoo.define('website.s_embed_code_options', function (require) {
+'use strict';
+
+const Dialog = require('web.Dialog');
+const core = require('web.core');
+const options = require('web_editor.snippets.options');
+
+const _t = core._t;
+
+options.registry.EmbedCode = options.Class.extend({
+    events: {
+        'click .o_we_edit_code': '_onEditClick',
+    },
+
+    _onEditClick: async function () {
+        const $container = this.$target.find('.o_embedded');
```

Comment:
Internal classes inside a snippet should be prefixed by the snippet class:

`s_embed_code_embedded` (seems a bit strange in this case but "o_embedded" is too generic).